Makefile

```diff
@@ -25,20 +25,20 @@ unregister-full:
 
 # only aware of aula sources
 sensei: .phony aula.unregister
-	$(EXEC) sensei $(AULA_SOURCES) tests/Spec.hs $(SENSEI_DEFAULT_ARGS) $(SENSEI_ARGS)
+	$(EXEC) sensei -j5 $(AULA_SOURCES) tests/Spec.hs $(SENSEI_DEFAULT_ARGS) $(SENSEI_ARGS)
```

Comment:
oops, sorry, i misunderstood your comment.

https://downloads.haskell.org/~ghc/latest/docs/html/users_guide/flag-reference.html#idp46760134970192

you sumbled over the fact that it's not a sensei flag, but a ghc flag.  (-:

(there is also something about cabal and -j, but i couldn't find a good reference for that.)

so, should I add a comment to the Makefile, or is it ok to expect people to find and read the ghc docs?